packages/aztec.js/src/proof/proofUtils.js

```diff
@@ -201,12 +201,68 @@ proofUtils.convertTranscript = (proofData, m, challengeHex, errors, proofType) =
 };
 
 /**
- * Generate a random note value that is less than K_MAX
+ * Override the behaviour of this method in the src folder.
```

Comment:
Can you add something about in which environments we override behaviour?

Reply:
Removed this comment that I added. It was left over from when I attempted something else and it doesn't add anything here. 

Should keep all local database initialisation comments/functionality to the `initDatabase.js` script